specification/security/resource-manager/Microsoft.Security/preview/2023-07-01-preview/integrations.json

```diff
@@ -0,0 +1,650 @@
+{
+	"swagger": "2.0",
+	"info": {
```

Comment:
>	 [](http://example.com/codeflow?start=0&length=1)

for cosistency , please use spaces instead of tabs in all files in this PR

Reply:
Not fixed. There still are a large number of tabs in this file. Please fix it before submitting